README.md

```diff
@@ -11,22 +11,22 @@ It consists of the following sections :
 
 Join Us
 -------
-Send us a mail to lyonytechhub@googlegroups.com or even request to join the group, it is totally free.
+It is totally free. Send us an mail at contact@lyontechhub.org, or [request access to our Slack](https://slack.lyontechhub.org).
 
-If you have a tech/IT community in Lyon area, request to join Lyon Tech Hub to promote your exciting activity to the community, it is simple as :
+If you have a tech/IT community in the Lyon area, and want to join Lyon Tech Hub to promote your exciting activity to the community, it is simple as:
 
 * Request access to Google Calendar
 * And submit a pull request to update web site (logo & JSON data).
 
-Make sure you do a pull request on master branch including the following :
+Make sure you do a pull request on master branch including the following:
 
 * your community in community list data/communities.json
 * your community details in file data/[key used in list].json
-* your community logo in imgs in PNG format with a 100x100 minimum resolution
+* your community logo in PNG format with a 100x100 minimum resolution

```

Comment:
we accept svg too, jpg doesn't work for some reason.

Reply:
because : https://github.com/lyontechhub/lyontechhub.github.io/blob/ad2b16b371c720ab4490202ed098c8ecddd82899/js/conferences/conferencesRepository.js#L4-L6